DSharpPlus/Clients/DiscordClient.cs

```diff
@@ -409,22 +399,42 @@ static void FailConnection(ManualResetEventSlim cl)
         }
     }
 
-    public Task ReconnectAsync(bool startNewSession = false)
-        => InternalReconnectAsync(startNewSession, code: startNewSession ? 1000 : 4002);
-
     /// <summary>
-    /// Disconnects from the gateway
+    /// Sends a raw payload to the gateway. This method is not recommended for use unless you know what you're doing.
     /// </summary>
-    /// <returns></returns>
-    public async Task DisconnectAsync()
+    /// <param name="opCode">The opcode to send to the Discord gateway.</param>
+    /// <param name="data">The data to deserialize.</param>
+    /// <remarks>
+    /// This method should not be used unless you know what you're doing. Instead, look towards the other 
+    /// explicitly implemented methods which come with client-side validation.
+    /// </remarks>
+    /// <returns>A task representing the payload being sent.</returns>
+    [Experimental("DSP0004")]
+    public async Task SendPayloadAsync(GatewayOpCode opCode, object? data)

```

Comment:
What about shard specific payloads? Voice ones are shard specific afaik

Reply:
we discussed this and i thought we arrived at payloads not being shard specific, would need testing. - shouldn't be too hard to add support for, though.

in related news, do we know whether shards have independent 120 payloads/min ratelimits? if so, that'd be something to expand on (probably in a future PR), properly balancing payloads
cc @VelvetToroyashi